libgnucash/backend/xml/io-gncxml-v1.cpp

```diff
@@ -472,6 +472,13 @@ simple_kvp_value_parser_new (sixtp_end_handler end_handler)
 
  */
 
+static gboolean
+string_to_gnc_numeric(const gchar* str, gnc_numeric *n)
+{
+    *n = gnc_numeric_from_string (str);
+    return (!gnc_numeric_check (*n));
+}
+
```

Comment:
Huh? It's not even used in io-gncxml-v1.

Reply:
It's used by the macro immediately afterwards

Reply:
Which is easily enough dealt with, but I'll accept that as out of scope for this PR.